coilsnake/modules/eb/EnemyModule.py

```diff
@@ -234,7 +234,7 @@ def read_from_project(self, resource_open):
     def upgrade_project(self, old_version, new_version, rom, resource_open_r, resource_open_w, resource_delete):
         if old_version == new_version:
             return
-        elif old_version == 3:
+        elif old_version <= 13:
```

Comment:
Please leave this `old_version == 3` block intact (without your `replace_field_in_yml` statements) and add a new block:
```py
elif old_version == 12:
    # Add your upgrade code here
    # replace_field_in_yml(...)
    # Continue upgrade chain
    self.upgrade_project(
        old_version + 1, new_version, rom, resource_open_r, resource_open_w, resource_delete)